src/game_ui.c

```diff
@@ -169,42 +173,42 @@ s32 D_800E283C[5] = {
 
 /************ .bss ************/
 
-u8 D_80126CD0;
-u8 D_80126CD1;
-u8 D_80126CD2;
-u8 D_80126CD3;
-u8 D_80126CD4;
-u8 D_80126CD5;
+s8 D_80126CD0;
+s8 D_80126CD1;
+s8 D_80126CD2;
+s8 D_80126CD3;
+s8 D_80126CD4;
+s8 D_80126CD5;
 s32 D_80126CD8;
 unk80126CDC *D_80126CDC;
-s32 D_80126CE0;
+unk80126CDC *D_80126CE0[1];
```

Comment:
**`D_80126CE0` to `D_80126CEC` are all unused. Should this be a bigger array?**

Reply:
I think there may be more than 3 indexes, but I'll set it to that, for the time being